### ISSUE TITLE
Scaling applied twice in SimilarityTransform?

### ISSUE THREAD

**Issue by taku-y** [BerkeleyAutomation/autolab_core#3]
https://github.com/BerkeleyAutomation/autolab_core/blob/cd3fa85e590fa8d3e86048d172d1321493ef09f2/autolab_core/rigid_transformations.py#L1004-L1006

https://github.com/BerkeleyAutomation/autolab_core/blob/cd3fa85e590fa8d3e86048d172d1321493ef09f2/autolab_core/rigid_transformations.py#L1045

It looks that the second one should be no scaling.

**Comment by jeffmahler** [comment]
You are right! I found a similar bug in the previous commit but missed this... I will fix tomorrow.
